parachain/runtime/tests/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 
 mod test_cases;
 
-use asset_hub_rococo_runtime::xcm_config::bridging::to_ethereum::BridgeHubEthereumBaseFeeInROC;
+use asset_hub_rococo_runtime::xcm_config::bridging::to_ethereum::DefaultBridgeHubEthereumBaseFeeInROC;
```

Comment:
Using the const version in the tests as the storage version causes it to panic due to there is no underlying storage provider.